lib/inspec/exceptions.rb

```diff
@@ -5,6 +5,8 @@ module Inspec
   module Exceptions
     class AttributesFileDoesNotExist < ArgumentError; end
     class AttributesFileNotReadable < ArgumentError; end
+    class ResourceFailed < StandardError; end
     class SecretsBackendNotFound < ArgumentError; end
+    class ResourceSkipped < StandardError; end
```

Comment:
Can we keep this list alphabetized for consistency?

Reply:
Hah! I originally had them called something else and took great care to make sure they were alphabetical. I renamed to both be `Resource` plus result so they would be together and forgot to sort.

Great catch @adamleff! Fixed in latest commit.